components/Announcements.js

```diff
@@ -1,70 +1,50 @@
 import React from 'react';
-import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
-
-const styles = StyleSheet.create({
-    rowContainer: {
-        flex: 1,
-        flexDirection: 'row',
-        borderBottomColor: 'gray',
-        borderBottomWidth: 1,
-        height: 150,
-    },
-    dateContainer: {
-        flex: 1,
-        flexDirection: 'column',
-    },
-    contentContainer: {
-        flex: 4,
-        flexDirection: 'column'},
-    date: {
-        fontWeight: 'bold',
-    },
-    title: {
-        color: 'blue',
-    },
-
-});
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
```

Comment:
It would have been better to do Announcements in a separate PR, actually 😅PRs are meant to keep changes modularized! If you don't already know how to undo/move to a new branch, it's ok! A bit too late now since we aren't meeting in person for awhile -- sometime I'll teach you some fun git stuff to do it

Reply:
Oh rip Im sorry :(

Reply:
It's ok! Just something to keep in mind for the future (: